pinot-segment-local/src/main/java/org/apache/pinot/segment/local/data/manager/TableDataManager.java

```diff
@@ -139,6 +139,14 @@ void addOrReplaceSegment(String segmentName, IndexLoadingConfig indexLoadingConf
    */
   List<SegmentDataManager> acquireAllSegments();
 
+  /**
+   * Acquires all segments of the table.
+   * <p>It is the caller's responsibility to return the segments by calling {@link #releaseSegment(SegmentDataManager)}.
+   *
+   * @return List of segment data managers
+   */
```

Comment:
minor again, want to introduce the param in doc above?

Reply:
Good catch, addressed.